src/app/nextversion/nextversion.go

```diff
@@ -17,10 +17,11 @@ import (
 )
 
 const (
-	tagPrefix   = "tag-prefix"
-	currentFlag = "current"
-	nextFlag    = "next"
-	gitRootFlag = "git-root"
+	outputPrefix = "output-prefix"
```

Comment:
name improvement are welcomed 😄

Reply:
since the command is next-version , i like this proposed name.